src/Router/BaseRouter.php

```diff
@@ -70,7 +70,7 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
             if ($this->user instanceof \LORIS\AnonymousUser) {
```

Comment:
@HenriRabalais do not acidentally merge this PR, this change in partiicular is only for cbigr open

Reply:
actually, @maltheism could you undo this ? i'll make this change on another PR just to cbigr-open

Reply:
@maltheism cant merge this PR if you keep this